commands/miscellaneous/hasRole.js

```diff
@@ -0,0 +1,57 @@
+const { MessageAttachment } = require('discord.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { ChannelType } = require('discord-api-types/v9');
+const { errorEmojiId } = require('../../config.json');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('hasrole')
+    .setDescription('Gets a list of user ids that were mentioned in a message and see if they are active or not.')
```

Comment:
Since the command is called "hasrole" and we're asking for a specific role to check, maybe we could change the descriptions and variable names and stuff into something generic? Instead of "activestudents" etc. something like "studentsWithRole" ? Plus we have a function in DiscordUtil "getUsersWithRoleFromServer(role, guild)" 😊

Reply:
Done! Thank you! 😊 It was actually a copy from the areactivestudents and forgot about that. 😅